lib/tasks/proxy.rake

```diff
@@ -33,11 +33,6 @@ namespace :proxy do
     end
   end
 
-  desc 'Fixing nil endpoint for hosted'
-  task :set_correct_endpoint_hosted => :environment do
-    Service.includes(:proxy).where(proxies: {endpoint: nil}, deployment_option: 'hosted').find_each(&:deployment_option_changed)
```

Comment:
this seems to be an old fix that no longer is necessary. Not necessary because such method does not exit since long time.